py/desitarget/sv1/sv1_cuts.py

```diff
@@ -1752,10 +1747,10 @@ def set_target_bits(photsys_north, photsys_south, obs_rflux,
     desi_target |= qsoz5_south * desi_mask.QSO_Z5_SOUTH
 
     # ADM add the per-bit information in the north for LRGs...
-    desi_target |= lrginit_n_4 * desi_mask.LRG_INIT_4PASS_NORTH
-    desi_target |= lrgsup_n_4 * desi_mask.LRG_SUPER_4PASS_NORTH
-    desi_target |= lrginit_n_8 * desi_mask.LRG_INIT_8PASS_NORTH
-    desi_target |= lrgsup_n_8 * desi_mask.LRG_SUPER_8PASS_NORTH
+    desi_target |= lrg_opt_n * desi_mask.LRG_OPT_NORTH
+    desi_target |= lrg_ir_n * desi_mask.LRG_IR_NORTH
+    desi_target |= lrg_sv_opt_n * desi_mask.LRG_OSV_PT_NORTH
```

Comment:
In the `py/desitarget/sv1/data/sv1_targetmask.yaml` file, you called this bit `LRG_SV_OPT_NORTH` (not `LRG_OSV_PT_NORTH`). I'm not sure why this isn't failing in your local tests, but unit tests caught it.

Reply:
Thanks. It's fixed.